Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -1765,43 +1730,42 @@ to `f x` in a neighborhood of `x`.-/
 theorem HasFiniteFPowerSeriesAt.eventually_const_of_bound_one
     (hf : HasFiniteFPowerSeriesAt f pf x 1) : f =ᶠ[𝓝 x] (fun _ => f x) :=
   Filter.eventuallyEq_iff_exists_mem.mpr (let ⟨r, hf⟩ := hf; ⟨EMetric.ball x r,
-  EMetric.ball_mem_nhds x hf.r_pos, fun y hy ↦ hf.eq_const_of_bound_one y hy⟩)
+    EMetric.ball_mem_nhds x hf.r_pos, fun y hy ↦ hf.eq_const_of_bound_one y hy⟩)
+
+section do_we_really_want_these_lemmas?
```

Comment:
These lemmas can be easily obtained using dot notation and may be removed

Reply:
It's true. I kept these lemmas (and wrote proofs that don't rely on the similar lemmas for analytic functions) because the proofs are mathematically much simpler in the finite case, so it bothered me to reduce to a more complicated statement. But if the proofs become just "use the statement for analytic functions", then I agree that the lemmas should be deleted.